blocks/api/test/serializer.js

```diff
@@ -226,7 +226,7 @@ describe( 'block serializer', () => {
 				''
```

Comment:
We might want to add tests for the non-`core/` namespaced case now.

Reply:
👍🏻